pkg/cloud-provider/cloudapi/aws/aws_security_test.go

```diff
@@ -323,9 +359,100 @@ var _ = Describe("AWS Cloud Security", func() {
 			err := cloudInterface.UpdateSecurityGroupRules(webSgIdentifier, addRule, []*securitygroup.CloudRule{}, addRule)
 			Expect(err).Should(BeNil())
 		})
+		It("Should fail to create egress rules", func() {
```

Comment:
what is generating fail case?? add a comment there too

Reply:
Add a comment explaining the test scenario.

Reply:
Done